Week1/Azad/searching_binary(0's & 1's).cpp

```diff
@@ -0,0 +1,23 @@
+# include<iostream>
+using namespace std;
+void swap(int &a,int &b){
+	int temp;
+	a=temp;
+	a=b;
+	b=temp;
+ }
+  int main(){
+	int n;
+	cin>>n;
+	int a[n];
+	for(int i=0;i<n; i++){
+        cin>>a[i];}	
+	for(int i=0; i<n-1;i++){
+	          for(int j=0; j<n; j++){
```

Comment:
you are having an o(n2) algorithm which is not desired, also this is bubble sort, you can do much better by doing a linear  search and just scanning the array once. i.e. in o(n) time